src/gtk/webview_webkit2.cpp

```diff
@@ -950,7 +950,11 @@ bool wxWebViewWebKit::IsBusy() const
 
 void wxWebViewWebKit::SetEditable(bool enable)
 {
+#if WEBKIT_CHECK_VERSION(2, 8, 0)
     webkit_web_view_set_editable(m_web_view, enable);
+#else
+    g_object_set(m_web_view, "editable", enable ? TRUE : FALSE, NULL);
```

Comment:
Thanks, I didn't notice this. Shouldn't we make `SetEditable()` return bool to indicate whether it managed to make the window editable? Using `IsEditable()` to check for the other function success looks a bit weird... But for now I guess I'll just make `IsEditable()` always return false for pre-2.8 versions.